src/editor/EditorStatusBar.js

```diff
@@ -37,9 +37,8 @@ define(function (require, exports, module) {
         AppInit                      = require("utils/AppInit"),
         DropdownButton               = require("widgets/DropdownButton").DropdownButton,
         EditorManager                = require("editor/EditorManager"),
-        MainViewManager     = require("view/MainViewManager"),
```

Comment:
Just as I saw that change, I think you can remove one level of indentation here.